README.md

```diff
@@ -85,6 +85,7 @@ config file like
 
 ```yaml
 contact_email: contact@company
+key_type: ec
```

Comment:
Should we default on `rsa` or `ec`? I think `rsa` is the most popular, perhaps could we put here `key_type: rsa # use "ec" to use a ECDSA key`

Reply:
indeed, example should have been the common use case